plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/BaseBigQueryConnectorTest.java

```diff
@@ -840,6 +840,20 @@ protected void verifyColumnNameLengthFailurePermissible(Throwable e)
                 .hasMessageContaining("Fields must contain only letters, numbers, and underscores, start with a letter or underscore, and be at most 300 characters long.");
     }
 
+    @Test
+    public void testCollate() {
+        String tableName = "test_collate" + randomNameSuffix();
+        try {
+            onBigQuery("CREATE TABLE test." + tableName + " AS (SELECT * FROM " +
+                    "UNNEST([COLLATE('B', 'und:ci'), 'b', 'a' ]) AS val order by val)");
```

Comment:
Please avoid using UNNEST. It's not intuitive and is unrelated to the test purpose.

Reply:
Got it, will try a simpler example. I just took the example which GCP has given in their [docs](https://cloud.google.com/bigquery/docs/reference/standard-sql/collation-concepts#collate_propagate).